app/models/nomis/sentence_detail.rb

```diff
@@ -27,6 +29,14 @@ def conditional_release_date
       @conditional_release_override_date.presence || @conditional_release_date
     end
 
+    def recall_release_date
+      if @actual_parole_date.presence && @actual_parole_date < @post_recall_release_override_date
```

Comment:
I think this comparison raises an error if `@actual_parole_date` is present but `@post_recall_release_override_date` is nil. 

It'd be good to test that case, and modify this to handle it gracefully.

Reply:
(although see my other comment - is `override_date` still meant to be part of this component?)